## 01_fix-stylesheet-path-for-debian.dpatch by Arjan Oosting <arjan@debian.org>

Index: c2hs-0.16.0/doc/Makefile
===================================================================
--- c2hs-0.16.0.orig/doc/Makefile
+++ c2hs-0.16.0/doc/Makefile
@@ -35,7 +35,7 @@
 	--param use.id.as.filename 1 \
 	--stringparam base.dir $(XSLTPROC_HTML_OUTDIR) \
 	--stringparam html.stylesheet $(XSLTPROC_HTML_CSS)
-XSLTPROC_HTML_STYLESHEET=http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl
+XSLTPROC_HTML_STYLESHEET=/usr/share/xml/docbook/stylesheet/nwalsh/xhtml/chunk.xsl
 XSLTPROC_OPTIONS=--nonet $(XSLTPROC_HTML_PARAMS) $(XSLTPROC_HTML_STYLESHEET)
 
 # build targets
